azext_iot/commands.py

```diff
@@ -188,6 +188,13 @@ def load_command_table(self, _):
             "validate-messages", "iot_central_validate_messages", is_preview=True
         )
 
+    with self.command_group(
+        "iot central app capability-model", command_type=iotcentral_ops
```

Comment:
Please remember to add the necessary help content for this command. Use the shared one for now.